easybuild/easyconfigs/c/CMake/CMake-3.8.1.eb

```diff
@@ -0,0 +1,24 @@
+# Contributed by Jean-Guillaume Piccinali and Luca Marsella (CSCS)
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = "3.8.1"
+
+homepage = 'http://www.cmake.org'
+description = """CMake, the cross-platform, open-source build system.
+ CMake is a family of tools designed to build, test and package software."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```

Comment:
I'd suggest to use: 
```
toolchain = {'name': 'dummy', 'version': ''}
```
since it could fail to load dependencies if the version 'dummy' is used.